docs/en/stack/ml/limitations.asciidoc

```diff
@@ -115,13 +115,13 @@ analyzed.
 
 
 [float]
-=== Time-based index patterns are not supported
+=== Time-based index patterns in {kib}
 //See x-pack-elasticsearch/#1910
 
-It is not possible to create a {ml} analysis job that uses time-based
-index patterns, for example `[logstash-]YYYY.MM.DD`.
-This applies to the single metric or multi metric job creation wizards in {kib}.
-
+If you try to create a {ml} analysis job that uses time-based index patterns
```

Comment:
I think this whole section can be dropped. I don't think it is even possible to create an index pattern like `[logstash-]YYYY.MM.DD` in the Kibana index patterns page. The docs only refer to using wlildcarded index patterns - see https://www.elastic.co/guide/en/kibana/current/tutorial-define-index.html.

Reply:
Thanks, I've removed that section entirely.